hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/parsing/Util.java

```diff
@@ -187,6 +189,60 @@ public static boolean isSegmentPresent(String v2Message, String segmentType) {
 		return false;
 	}
 	
+	/**
+	 * @param v2Message
+	 * @param segmentType
+	 * @return
+	 */
+	public static String getZCBSegment(String v2Message, String segmentType) {
+		String[] v2DataLines_Pharmanet = v2Message.split(LINE_BREAK);
```

Comment:
Java variable name conventions would be v2DataLinesPharmanet instead of v2DataLines_Pharmanet

Reply:
done